module/applications/actor/base-sheet.mjs

```diff
@@ -1233,7 +1299,7 @@ export default class ActorSheet5e extends ActorSheet {
     const itemData = {
       name: game.i18n.format("DND5E.ItemNew", {type: game.i18n.localize(`ITEM.Type${type.capitalize()}`)}),
       type: type,
-      system: foundry.utils.expandObject({ ...header.dataset })
+      system: foundry.utils.expandObject(dataset)
```

Comment:
Fairly sure this doesn't work because `dataset` is actually a `DOMStringMap` and not a plain object, so this needs to be:

```suggestion
      system: foundry.utils.expandObject({ ...dataset })
```